src/main/java/vazkii/botania/common/item/relic/ItemLokiRing.java

```diff
@@ -170,43 +194,78 @@ public void onPlayerInteract(PlayerInteractEvent event) {
 			player.posZ = oldPosZ;
 		}
 	}
+	
 	public static void setMode(ItemStack stack, boolean state) {
 		stack.stackTagCompound.setBoolean(TAG_MODE, state);
 	}
+
+	public static void setBreakingMode(ItemStack stack, boolean state) {
+		stack.stackTagCompound.setBoolean(TAG_BREAKING_MODE, state);
+	}
+
+
 	@SideOnly(Side.CLIENT)
-	public static void renderHUDNotification(){
+	public static void renderHUDNotification(String type){
 		Minecraft mc = Minecraft.getMinecraft();
-		String text = getLokiModeText(getLokiRing(mc.thePlayer));
+		String text;
+		switch (type) {
+			case  ("mode"):
+				text = getLokiModeText(getLokiRing(mc.thePlayer));
+				break;
+			case ("breaking"):
+				text = getLokiBreakingModeText(getLokiRing(mc.thePlayer));
+				break;
+			default:
+				return;
+				
+		}
+		 
 		GTNHLib.proxy.printMessageAboveHotbar(text, 60, true, true);
 	}
 	public static String getLokiModeText(ItemStack stack){
 		return EnumChatFormatting.GOLD + StatCollector.translateToLocal("item.botania:lokiRing.name") + " " + (isRingEnabled(stack) ?
 				EnumChatFormatting.GREEN + StatCollector.translateToLocal("botaniamisc.lokiOn") :
 				EnumChatFormatting.RED + StatCollector.translateToLocal("botaniamisc.lokiOff"));
 	}
+
+	public static String getLokiBreakingModeText(ItemStack stack){
+		return EnumChatFormatting.GOLD + StatCollector.translateToLocal("item.botania:lokiRing.name") + " "+
+				StatCollector.translateToLocal("botaniamisc.breaking") + " " + (isRingBreakingEnabled(stack) ?
+				EnumChatFormatting.GREEN + StatCollector.translateToLocal("botaniamisc.lokiOn") :
+				EnumChatFormatting.RED + StatCollector.translateToLocal("botaniamisc.lokiOff"));
+	}
+
 	public static boolean isRingEnabled (final ItemStack stack){
 		return stack.stackTagCompound.getBoolean(TAG_MODE);
 	}
+
+	public static boolean isRingBreakingEnabled (final ItemStack stack){
+		return stack.stackTagCompound.getBoolean(TAG_BREAKING_MODE);
+	}
+
 	public static void breakOnAllCursors(EntityPlayer player, Item item, ItemStack stack, int x, int y, int z, int side) {
 		ItemStack lokiRing = getLokiRing(player);
-		if(lokiRing == null || player.worldObj.isRemote || !(item instanceof ISequentialBreaker))
+		if(lokiRing == null || player.worldObj.isRemote || !isRingEnabled(lokiRing) || !isRingBreakingEnabled(lokiRing))
 			return;
-
+		boolean isISequentalBreaker = item instanceof ISequentialBreaker;
```

Comment:
Variable can be removed; testing for `breaker != null` instead makes the code shorter and the nullability correctness easier to demonstrate.